.project_automation/functional_tests/entrypoint.sh

```diff
@@ -23,13 +23,46 @@ cleanup_all_regions() {
     done
 }
 
+run_scoutsuite() {
+    #Create Scoutsuite security scan custom rule
+    python3 .project_automation/functional_tests/create-scoutsuite-custom-rule.py
+    # Execute Scoutsuite security scan
+    scout aws -r us-east-1 --ruleset .project_automation/functional_tests/abi-scoutsuite-custom-ruleset.json --no-browser --max-rate 5 --max-workers 5
+    # Upload Scoutsuite security scan results to S3 bucket named scoutsuite-results-aws-AWS-ACCOUNT-ID
+    python3 .project_automation/functional_tests/process-scoutsuite-report.py
+    # Delete taskcat e2e test resources
+    taskcat test clean ALL
+}
+
+process_scoutsuite_report() {
+    # Check Scoutsuite security scan result for Danger level findings (Non-0 exit code)
+    scoutsuite_sysout_result=$(cat scoutsuite_sysout.txt)
+    #rm scoutsuite_sysout.txt
+    if [ "$scoutsuite_sysout_result" -ne 0 ]; then
+        # The value is non-zero, indicating Scoutsuite report needs to be checked for security issues
+        echo "Scoutsuite report contains security issues. Details below or check the S3 bucket named scoutsuite-results-aws-AWS-ACCOUNT-ID in the test account."
+        
+        # Retrieve the AWS account ID and store it in a variable
+        AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
```

Comment:
Can we move this to the top outside the function? I believe there is no need to run this command multiple times.

Reply:
Moved to the top outside of the function.